package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "landuseplanning-api",
-  "version": "1.5.1",
+  "version": "1.5.2",
```

Comment:
i'm pretty sure this is more than a patch.. 
Could you use your old configurations, and this would work.. Probably not.. One might even argue its major version, but i still believe it is only a minor.

Reply:
I was thinking about this from the user's(to them nothing has changed and this is "maintenance") perspective but maybe that's the wrong approach.

I guess from a developer's perspective it is a breaking change as far as app config goes...